### PR TITLE
adding tiered storage confs to endpoint

### PR DESCRIPTION
**Pull request by jerrypeng**
add tiered storage configuration information to the internal-confs endpoint so broker confs for tiered storage can be retrieved

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java [hunk 1]
```diff
@@ -209,7 +210,7 @@ void internalConfiguration() throws Exception {
         InternalConfigurationData expectedData = new InternalConfigurationData(
```
Comment (sijie) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java]: Does this have any credential information?

File: pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java [hunk 2]
```diff
@@ -58,7 +66,8 @@ public boolean equals(Object obj) {
         InternalConfigurationData other = (InternalConfigurationData) obj;
```
Comment (merlimat) [inline pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java]: We could just use lombok to generate equals and toString methods

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java (hunk 1 above)
Comment (jerrypeng) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java]: its does not contain the keys

File: pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java (hunk 2 above)
Comment (jerrypeng) [reply pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java]: changed to using lombok

**Comment by jerrypeng** [comment]
rerun java8 tests

**Comment by jerrypeng** [comment]
rerun integration tests

**Comment by jerrypeng** [comment]
rerun java8 tests

**Comment by jerrypeng** [comment]
rerun java8 tests

**Comment by jerrypeng** [comment]
rerun java8 tests

**Comment by jerrypeng** [comment]
rerun java8 tests

**Comment by jerrypeng** [comment]
rerun java8 tests

**Comment by sijie** [comment]
this PR needs to rebase to master. there is a unit test fix on NonPersistentTopicTest and also there are renames on tiered storage classes

**Comment by sijie** [comment]
@jerrypeng I have sent out #2398 for review. I think this one can be closed in favor of using #2398

**Comment by sijie** [comment]
This is not needed any more.

**Pull request closed without merging**